src/ipv4/fragments.ml

```diff
@@ -180,3 +180,38 @@ let process cache ts (packet : Ipv4_packet.t) payload =
           | Error Hole -> maybe_add_to_cache cache', None
         else
           maybe_add_to_cache cache', None
+
+(* TODO hdr.options is a Cstruct.t atm, but instead we need to parse all the
+   options, and distinguish based on the first bit -- only these with the bit
+   set should be copied into all fragments (see RFC 791) *)
```

Comment:
Would it make sense with a more specific reference? RFC 791 is kind of long, and shows up with a bunch of "Updated by" and "Errata Exist".

Reply:
tbh, if modifying this code it is best to read 791 _and_ _all_ updates _and_ _all_ errata. I put the section 3.1 in the comment, and now refer to page 15 which includes the description of `copied`.